App.js

```diff
@@ -1,13 +1,29 @@
 import React from 'react'
 import { StatusBar } from 'expo-status-bar'
 import { Provider as PaperProvider } from 'react-native-paper'
+import {
+  useFonts as usePoppins,
+  Poppins_700Bold,
+} from '@expo-google-fonts/poppins'
+import {
+  useFonts as useDidactGothic,
+  DidactGothic_400Regular,
+} from '@expo-google-fonts/didact-gothic'
 
 import { AuthenticationProvider } from './context/authentication'
 import { SecretsProvider } from './context/secrets'
 import Main from './components/Main'
 import theme from './lib/defaultTheme'
 
 export default function App() {
+  usePoppins({
+    Poppins_700Bold,
+  })
+
+  useDidactGothic({
+    DidactGothic_400Regular,
+  })
```

Comment:
you have to wait until the fonts are loaded. see the docs

Reply:
Yep 🤦 . Done https://github.com/nearform/optic-expo/pull/21/commits/6974fcc9fa6692c17967c246fb4aa0e9260ed39a